doc/user/layouts/partials/toc.html

```diff
@@ -93,11 +93,7 @@ <h2 class="gradient_text dark">On this page</h2>
   // Scroll to our massaged offsets on a hashchange event, to override the
   // browser's automatic scroll to the unmassaged offset.
   $(window).on("hashchange", function () {
-    const id = window.location.hash.substring(1);
-    const offsets = headingOffsets();
-    const offset = offsets.find((o) => o.id == id);
-    if (offset) $(window).scrollTop(offset.offset - 10);
-    else $(window).trigger("scroll");
+   $(window).trigger("scroll");
```

Comment:
Had to tweak this -- otherwise, it wouldn't scroll correctly to the bottom.   The rewrite of the scroll logic is on my todo list - it's the highest of my non-content items.  Fingers crossed that maybe in December.